pset-functions.js

```diff
@@ -31,6 +31,32 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+
+
+function blackJack(playerCardScore, dealerCardScore){
+  let result = 0;
+  if (playerCardScore > 21 && dealerCardScore > 21) {
+    return 0
+  } else if (playerCardScore > 21) {
+    return dealerCardScore
+  } else if (dealerCardScore > 21) {
+    return playerCardScore
+  } else if (playerCardScore === dealerCardScore) {
+    return playerCardScore
+  } else if (playerCardScore > dealerCardScore) {
+    return playerCardScore
+  } else if (playerCardScore < dealerCardScore) {
+    return dealerCardScore
+  } else {
+    return 'Not valid number'
+  }
+}
+
+console.log(blackJack(19, 21))
+console.log(blackJack(22, 22))
+console.log(blackJack(19, 22))
+console.log(blackJack(21, 21))
+
```

Comment:
@rlou88 Good work here writing the`blackjack` function. I see you're utilizing the conditionals in a good way to check to see the different winning outcomes for the game.

Take a look at the solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L30)